src/align.rs

```diff
@@ -141,6 +141,15 @@ pub struct AlignedVec<T: Copy, C: AlignedByteChunk> {
 }
 
 impl<T: Copy, C: AlignedByteChunk> AlignedVec<T, C> {
+    // Note that in Rust, no single allocation can exceed `isize::MAX` __bytes_.
+    const MAX_LEN: usize = {
+        if core::mem::size_of::<T>() == 0 {
```

Comment:
```suggestion
        if mem::size_of::<T>() == 0 {
```

We already have `use std::mem` and use `mem::size_of` of elsewhere.